Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -334,7 +322,8 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
 - (CGSize)sizeWithFont:(UIFont*)font constrainedToSize:(CGSize)size lineBreakMode:(UILineBreakMode)lineBreakMode {
     return [self _sizeWithAttributes:@{
         NSFontAttributeName : font,
-        NSParagraphStyleAttributeName : _paragraphStyleWithLineBreakMode(lineBreakMode)
+        static_cast<NSString*>(kCTParagraphStyleAttributeName) :
+            (id)[_paragraphStyleWithLineBreakMode(lineBreakMode) _createCTParagraphStyle]
```

Comment:
This creator returns a +1; you'll need to make sure it's appropriately lifetime-tracked here.